test/functional/terminal/scrollback_spec.lua

```diff
@@ -416,7 +438,11 @@ describe("'scrollback' option", function()
     -- Terminal job data is received asynchronously, may happen before the
     -- 'scrollback' option is synchronized with the internal sb_buffer.
     command('sleep 100m')
-    feed_data('for i in $(seq 1 40); do echo "line$i"; done\n')
+    if iswin() then
+      feed_data('for($i=1;$i -le 40;$i++){Write-Host \"line$i\"}\r')
```

Comment:
No, it's left hanging, waiting for more input (`>>` appears).

`\r` forces the command to run.
```viml
:call jobsend(b:terminal_job_id, "\r")
```

Seems `` `r `` and `` `n `` works only for multi-line output in `Write-Host`, `Write-Output`, etc.